src/intTest/java/uk/nhs/digital/nhsconnect/nhais/jms/InboundMeshServiceRegistrationTest.java

```diff
@@ -15,12 +16,14 @@
 import uk.nhs.digital.nhsconnect.nhais.service.TimestampService;
 import uk.nhs.digital.nhsconnect.nhais.utils.OperationId;
 
-import javax.jms.JMSException;
-import javax.jms.Message;
-import java.io.IOException;
-import java.nio.file.Files;
-import java.time.Instant;
-import java.time.ZonedDateTime;
+import org.assertj.core.api.SoftAssertions;
```

Comment:
Why? Are we all using the same IntelliJ reformat rules?

Reply:
Seems so. Do we have any standard for that defined?

Reply:
We've been using default Intellij. Can you check it with the other PR I've created? https://github.com/nhsconnect/integration-adaptor-nhais/pull/88
Just check if having this does change anything in code reformat